Yoast/wordpress-seo
# P3 121 create tests for modal components

## Context


* We want to test our SEMrush modal components so we can assure they keep working as expected.

## Summary


This PR can be summarized in the following changelog entry:
* Implements tests for `js/src/components/modals/SEMrushCountrySelector.js` and `js/src/components/modals/SEMrushKeyphrasesTable.js` to test their functionality.

## Relevant technical choices:

* At first, i wanted to test a change within the select element for the country selector, however me and @bintzandt discovered that with our current implementation of select2 and jquery via window and wordpress this is nearly impossible to do. That being said, he and @karlijnbok have already been looking to replace select2 in the future so we agreed that this is not needed for now.
* I did not write tests for the following files since they are too small to usefully test: `SEMrushLimitReached`, `SEMrushLoading`, `SEMrushMaxRelatedKeyphrases`, `SEMrushRequestFailed` and `SEMrushUpsellAlert`.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and build
* Run `yarn test` and ensure they all pass

## UI changes

## Documentation

## Quality assurance
